packages/ui/tsconfig.json

```diff
@@ -16,8 +16,9 @@
     "jsx": "preserve",
     "incremental": true,
     "downlevelIteration": true,
-    "composite": true
+    "composite": true,
+    "sourceMap": true,
```

Comment:
trailing commas are invalid in json files, please do remove them here and below!